src/views/tickets/components/Session.vue

```diff
@@ -134,7 +134,7 @@ export default {
     },
     onMonitor() {
       const joinUrl = `/luna/monitor/${this.session.id}?ticket_id=${this.object.id}`
-      window.open(joinUrl, 'height=600, width=850, top=400, left=400, toolbar=no, menubar=no, scrollbars=no, location=no, status=no')
+      window.open(joinUrl, '_blank', 'height=600, width=850, top=400, left=400, toolbar=no, menubar=no, scrollbars=no, location=no, status=no')
     },
     onToggleLock() {
       const url = '/api/v1/terminal/tasks/toggle-lock-session-for-ticket/'
```

Comment:
There do not seem to be any significant code differences detected that could indicate an issue or need for improvement. The provided code appears correct and follows the guidelines and standards established at the knowledge cutoff (September 1st, 2021) when using the Vue.js library for web application development.